projects/ion/src/lib/tag/tag.component.ts

```diff
@@ -38,4 +38,18 @@ export class TagComponent {
   getTagColor() {
     return validateHexColor(this.color) ? this.color : defaultColor;
   }
+
+  hasLabel() {
+    return !this.label || this.label.trim() === '';
+  }
+
+  validateLabel() {
+    if (this.hasLabel()) {
+      throw new Error('Invalid Tag label informed');
+    }
+  }
+
+  ngOnInit(): void {
+    this.validateLabel();
```

Comment:
Hmmm I think we have a problem putting this validation only at onInit().
If I render <ion-tag label="someVariable"> with someVariable = 'label', and then change someVariable to "        ", tag would not emit any error and will appear empty. ([Here you can see a demonstration of what I'm talking about.](https://user-images.githubusercontent.com/57760325/191968780-72cd6d7e-7931-4428-8dba-0eb076bc72cc.webm))
You can validate with @edneyql or @iurynogueira if this is really a problem or no.

Reply:
I think this can be solved with ngOnChanges.

Reply:
I think that a problem